### ISSUE TITLE
[Bug Report] Tooltip doesn't work on touch screen

### ISSUE THREAD

**Issue by Aukemid** [vuetifyjs/vuetify#12279]
### Environment
**Vuetify Version:** 2.3.10
**Vue Version:** 2.6.12
**Browsers:** Other
**OS:** Android, iOS

### Steps to reproduce
https://vuetifyjs.com/en/components/tooltips/
The usage examples in the docs for the Tooltip component work correctly in all my tests. If you follow the given codepen, from that usage example, it doesn't work as it should and differently between iOS and Android. The codepen in the reproduction link is the same as the codepen from the docs.

### Expected Behavior
The tooltip should work for the button, icon and text on all devices.

### Actual Behavior
On iOS the tooltip only works for the button, not for the icon and text. Tested on iPhone and iPad. In an Android test the tooltip worked for the icon, but not for the button and text.

### Reproduction Link
<a href="https://codepen.io/Aukemid/pen/BaKMWGa" target="_blank">https://codepen.io/Aukemid/pen/BaKMWGa</a>

### Other comments
There are multiple similar open issues for tooltips on touch devices, but those describe more specific situations. This seems to be a more generic problem with the tooltip component, but the cause might be the same.

**Comment by KaelWD** [comment]
Android: button works on long press, icon and text work on either tap or long press

iOS: only works on tap for all three

**Comment by Aukemid** [comment]
> Android: button works on long press, icon and text work on either tap or long press
> 
> iOS: only works on tap for all three

On Android it seems to work as you described, but I can't get it to work on iOS. I've tested this on an iPhone and iPad and had 4 colleagues test it on there iPhones.

**Comment by KaelWD** [comment]
The only iOS device I have is an iPad running iOS 12.4.2. I just tried an iPhone 11 on browserstack and that worked too.

**Comment by Aukemid** [comment]
All my iOS tests were on 13.7 and 14.0.

**Comment by Aukemid** [comment]
I've updated the codepen in the reproduction link. The tooltip on the icon and text in the second row do work because of the added @click event. However, the test method the click event should call is not executed when clicking the icon or text.

**Comment by johnleider** [comment]
Duplicate of #7114

If you have any additional questions, please reach out to us in our [Discord community](https://community.vuetifyjs.com).

**Comment by Jonathas-Augusto** [comment]
I know it's been years, but I wanted to leave a note for those who work with old applications and need to make the tooltip work on mobile. Just add `retain-focus-on-click` props in your v-btn and it will work

**Comment by Tex-Hex** [comment]
UPDATE in 06-2024 (if anyone stumple upon the same question):
As the property `retain-focus-on-click` has been removed in Vuetify 3 (see https://vuetifyjs.com/en/getting-started/upgrade-guide/), we could solve the issue by using https://vuetifyjs.com/en/api/v-tooltip/#props-open-on-click:
So in the end we have:
```JavaScript
<v-tooltip open-on-click="true"...>
```
As `open-on-hover` default to true, this will work on desktop.
And the `open-on-click` option is working fine for mobile.
Hope the same applies for you...

**Comment by mikhaylov-ya** [comment]
Tooltip doesn't work on long tap on mobile devices with `open-on-click`. It opens only after `touchleave`. Is there a genuine way to enable mobile tooltips (on buttons) on long tap without finished click? There is a way to do it with `v-touch` but in this case I should provide `v-model` for each tooltip
